### PR TITLE
Install Protobuf header and FPGA fixes

### PR DESCRIPTION
**Pull request by n-eiling**

### CODE REVIEW

**Comment by stv0g** [comment]
Looks good to me overall. I left a comment about the install location of format headers.

Another small nit: I try using [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) in VILLAS-related projects. While this standard does not mandate how the description should be capitalized, I see more often that people start the description with an upper case letter.

Can we agree on starting commit descriptions with an upper case letter?
Example:

```diff
-fpga: make FPGA support sending and receiving integers
+fpga: Make FPGA support sending and receiving integers
```

File: lib/formats/CMakeLists.txt [hunk 1]
```diff
@@ -41,6 +41,15 @@ if(DEFINED PROTOBUFC_COMPILER AND PROTOBUFC_FOUND)
     list(APPEND LIBRARIES
         PkgConfig::PROTOBUFC
     )
+
+    install(
+        DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+        DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/villas
```
[the remark this answers is not in the document]
Comment (stv0g) [reply lib/formats/CMakeLists.txt:47]: I agree. I think the protobuf header should be located at `include/villas/formats/villas.pb-c.h`

[the remark this answers is not in the document]
Comment (n-eiling) [reply lib/formats/CMakeLists.txt]: i think this is it: https://github.com/VILLASframework/node/pull/806/commits/a01e0239be38c9882cdd4667d8665cd87532c1f2

**Comment by n-eiling** [comment]
It's sooo difficult to change this habit :) I'll try to remember.

File: lib/formats/CMakeLists.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (n-eiling) [reply lib/formats/CMakeLists.txt]: ok now it works. I didn't get a separate target / task to work so I just added the copy operation to the existing protobuf compiler task.

[the remark this answers is not in the document]
Comment (stv0g) [reply lib/formats/CMakeLists.txt]: :D Just wondering: why didnt you modify just the output path of the protobuf compiler invocation? This is fine with me. But maybe we could have eliminated this extra copy?

**Review by stv0g — APPROVED** [review]

**Pull request merged**